package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "on-chain-identity-gateway",
+    "version": "1.0.0",
+    "description": "Welcome to the Identity.com On-chain Identity Gateway monorepo.",
+    "main": "index.js",
+    "directories": {
+        "doc": "docs"
+    },
+    "devDependencies": {
+        "husky": "^8.0.1",
+        "npm-run-all": "^4.1.5"
+    },
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "prepare": "husky install",
+        "lint": "cd solana/program_v2 && yarn lint"
```

Comment:
I feel the root-level package json should ONLY contain husky related stuff and link out to other projects at only a single point.

e.g. if we want to add the another folder to be linted now how would we add it?

Reply:
> We are using `yarn` and not `npm`, correct. Therefore I'd expect a `yarn.lock` and not a package-lock.json.

changed npm to yarn